### PR TITLE
TDD for issues 944, 945, 946

### PR DESCRIPTION
**Pull request by Moosh-be**
I try  to write a test wich check  issues 
- [x] #944 GE IBAN
- [x] #945 PK IBAN 
- [x] #946 TR IBAN
are solved.

See, it's not.

And after solve them
- [ ] #944 GE IBAN
- [ ] #945 PK IBAN 
- [ ] #946 TR IBAN

But i'm too bad to check  a string is a "number"

I continue to learn.

All feedback to better write this is welcome
 
!!! **This PR contain actually only the test, not the solution for issues**

### CODE REVIEW

**Comment by Moosh-be** [comment]
Need a Review of the test. 
I've create a "function" isNumeric, just beacause I Don't found how to do it with chai.

File: test/finance_issue.unit.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+if (typeof module !== 'undefined') {
+    var assert = require('assert');
+    var faker = require('../index');
+}
+function IsNumeric(input)
```
Comment (Marak) [inline test/finance_issue.unit.js:5]: Can we get a more standard `IsNumeric` check here? We may not need a custom fn.

I would recommend spending five minutes looking into how JS handles numbers.

I think `Number.isFinite` might work here? I'm not sure.

Comment (Moosh-be) [reply test/finance_issue.unit.js:5]: Locally tested.  It's work fine with `Number.isFinite` But I'm fighting  with git rebase and conflict ;(

I continue to fight ;-)

**Comment by Moosh-be** [comment]
not mergable !!   wait fix  for dog fix

**Comment by Marak** [comment]
@Moosh-be - 

What happened here? I don't think you need to be co-authoring 188 commits.

I would recommend trying a new PR.

**Comment by Moosh-be** [comment]
> I would recommend trying a new PR.

I agree...... 


I hate Git ;) (or I'm just too stupid to understand how it's works)

**Comment by Moosh-be** [comment]
I spend 90% of my time to understand to use git.... 10% to contribute...  It's boring.

**Comment by Moosh-be** [comment]
I'm back with #1112

**Comment by Moosh-be** [comment]
@Marak  the label "Will Merge - Needs Works"  can be removed, I never reopen this PR, I've restart and complete in #1112

**Pull request closed without merging**